labs/lab08-airgap-lite/README.md

```diff
@@ -1,72 +1,67 @@
 Lab 1.8: Airgap
 =========================================
 
-In this lab, we'll review how to perform installations in Air Gap environments, and
-how to collect support bundles in Air Gap environments.
+In this lab, you will review how to perform installations and collect support bundles in air gap environments.
 
 * **What you will do**:
-    * Access and verify a single-node Air Gap setup via a bastion server
-    * Learn to use KOTS to install in an Air Gap environment
-    * Create an SSH tunnel to configure an Air Gap instance
-    * Perform an upgrade of an application in an Air Gap environment
-    * Use the `kubectl support-bundle` CLI in an Air Gap environment
-* **Who this is for**: This lab is for anyone who builds/maintains KOTS applications (see note below)
+    * Access and verify a single node air gap setup using a bastion server
+    * Learn to use app manager to install in an air gap environment
+    * Create an SSH tunnel to configure an air gap instance
+    * Perform an upgrade of an application in an air gap environment
+    * Use the `kubectl support-bundle` CLI in an air gap environment
+* **Who this lab is for**:
+    * Anyone who builds or maintains app manager applications (see note below)
     * Full Stack / DevOps / Product Engineers
 * **Prerequisites**:
     * Basic working knowledge of Kubernetes
 * **Outcomes**:
-    * You will be ready to deliver a KOTS application into an Air Gap environment
-    * You will build confidence in performing upgrades and troubleshooting in Air Gap environments
+    * You will be ready to deliver a app manager application into an air gap environment
```

Comment:
Since this is a more general audience, what do y'all think about trying to downplay the Replicated product verbiage here where its not crucial to understanding

```suggestion
    * You will be ready to deliver a Kubernetes application into an air gap environment
```